django/db/models/sql/constants.py

```diff
@@ -9,9 +9,15 @@
 # Namedtuples for sql.* internal use.
 
 # How many results to expect from a cursor.execute call
+# multiple rows are expected
 MULTI = "multi"
+# a single row is expected
 SINGLE = "single"
+# do not return the rows, instead return the cursor
+# used for the query
 CURSOR = "cursor"
+# instead of returning the rows, return the row count
+ROW_COUNT = "row count"
 NO_RESULTS = "no results"
```

Comment:
```suggestion
MULTI = "multi"
SINGLE = "single"
NO_RESULTS = "no results"
# Rather than returning rows, returns:
CURSOR = "cursor"
ROW_COUNT = "row count"
```